### PR TITLE
player icon and logging

### PR DESCRIPTION
**Pull request by Flippchen**
- changed player icon
- changed borders for player

### CODE REVIEW

File: game/Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,12 @@ serde = "1.0.145"
 serde_yaml = "0.9.13"
 ggez = "0.8.1"
 dyn-clone = "1.0.9"
+tracing = "0.1.37"
+tracing-subscriber = "0.3.16"
+tracing-appender = "0.2"
 fastrand = "1.8.0"
+chrono = "0.4.23"
+tempfile = "3.3.0"
```
Comment (MarionHinkel) [inline game/Cargo.toml]: Sieht unnötig aus

File: game/src/backend/screen.rs [hunk 2]
```diff
@@ -3,13 +3,14 @@ use crate::backend::utils::get_scale;
 use crate::error::RLError;
 use crate::main_menu::mainmenu::MainMenu;
 use crate::{draw, RLResult};
-use ggez::conf::FullscreenType::True;
```
Comment (MarionHinkel) [inline game/src/backend/screen.rs:6]: wieder rein den kram

**Review by MarionHinkel — APPROVED** [review]
LGTM

**Review by mfloto — APPROVED** [review]
cargo fix does a lot...
LGTM

**Review by SanderForGodot — APPROVED** [review]

**Pull request merged**